packages/documentation/pages/utilities.ts

```diff
@@ -69,6 +74,47 @@ export const createActions = (
 		  ]
 		: undefined
 
+export const createRemoteUpload = (
+	hasRemoteUpload: boolean,
```

Comment:
honestly doesn't feel like the arg is needed , you already check for the component name before calling this function

Reply:
Also needed bc it's called internally by `generateComponentCode` to generate the component's code.